requirements.md

```diff
@@ -0,0 +1,147 @@
+# Requirements for Presentation API protocols 
+
+This document is a working draft that captures a set of technical requirements
+for the network protocols to be developed by the [Second Screen Community
+Group](https://webscreens.github.io/cg-charter/).  These requirements are
+intended to guide the development of the specifications to be developed by the
+Second Screen Community Group.
+
+The terms *controlling user agent*, *receiving user agent*, *controlling
+browsing context*, *receiving browsing context*, and *destination browsing
+context* are defined in the [Presentation
+API](https://w3c.github.io/presentation-api/).
+
+Below, the term "controller" is used as short for a controlling user agent and
+"receiver" as short for a receiving user agent.  "Presentation display" refers
+to a network-connected device that hosts the receiving user agent.
+
+## <a name="spec-ddc"></a>Presentation Display Discovery and Connection requirements
+
+The following requirements apply to the proposed specification, *Presentation
+Display Discovery and Connection*.
+
+### <a name="REQ-D01"></a>REQ-D01: Presentation Display Availability
+
+- A controller shall be able to discover the presence of a receiver connected to
+the same IPv4 or IPv6 subnet and reachable by IP multicast.
```

Comment:
I'm adding some deliverables to the charter for non-normative reports on other technologies for discovering nearby screens (bluetooth, NFC).  I can add a note here that requirements for these technologies are to be determined (and will be embedded within those reports).